esbuild.mjs

```diff
@@ -0,0 +1,15 @@
+import * as esbuild from 'esbuild'
+
+await esbuild
+  .build({
+    entryPoints: ['./app/receiver/index.ts'],
+    bundle: true,
+    outfile: '/receiver/index.js',
+    platform: 'node',
+  })
+  .then(() => {
+    console.log('done')
+  })
+  .catch((e) => {
+    console.log(e)
+  })
```

Comment:
Won't the error message be printed regardless?

```suggestion
```

Reply:
Fair enough, I'll remove this. It was just copied boilerplate stuff